audio_service/ios/Classes/AudioServicePlugin.h

```diff
@@ -35,10 +35,10 @@ enum MediaAction {
     APrepareFromSearch,
     APrepareFromUri,
     ASetRepeatMode,
-    AUnused_1, // deprecated (setShuffleModeEnabled)
-    AUnused_2, // setCaptioningEnabled
+    ASetCaptioningEnabled,
```

Comment:
This shouldn't be changed because the position in the enum corresponds to Android's `PlaybackStateCompat.ACTION_*` constants.

Reply:
This matches the dart side so let's discuss it there